apps/deploy-web/src/pages/api/healthz.ts

```diff
@@ -0,0 +1,5 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
```

Comment:
this is a formal endpoint to be used in k8s. we might add something more meaningful here if there is anything (like db check)

Reply:
ok, I see not that the path is broken there. anyway, i couldn't spot how it affects ui so just dropped it